src/AppInstallerCLICore/ExecutionReporter.h

```diff
@@ -165,9 +167,14 @@ namespace AppInstaller::CLI::Execution
             m_progressSink = sink;
         }
 
+        void SetLevelMask(Level reporterLevel, bool setEnabled = true);
+
+        Level GetLevelMask() {

```

Comment:
nit: we would not publicly expose internal flags. Use the method IsLevelEnabled() in the previous iteration and let caller call it.